Helper/DonationsHelper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Adyen\Payment\Helper;
+
+use Adyen\Model\Checkout\DonationCampaignsRequest;
+use Adyen\Payment\Logger\AdyenLogger;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Adyen\Payment\Helper\Data;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+class DonationsHelper extends AbstractHelper
+{
+    private Data $adyenHelper;
+
+    /**
+     * @var Config
+     */
+    private Config $configHelper;
+    /**
+     * @var AdyenLogger
+     */
+    protected $adyenLogger;
+
+    public function __construct(
+        Context $context,
+        Data $adyenHelper,
+        Config $configHelper,
+        AdyenLogger $adyenLogger
+    ) {
+        parent::__construct($context);
+        $this->adyenHelper = $adyenHelper;
+        $this->configHelper = $configHelper;
+        $this->adyenLogger = $adyenLogger;
+    }
+
+    /**
+     * @throws NoSuchEntityException
+     * @throws LocalizedException
+     */
+    public function fetchDonationCampaigns(array $payloadData, int $storeId): array
```

Comment:
Btw, this endpoint can be simplified to always return 1 or 0 campaign details instead of an array. 
`fetchDonationCampaign(): {}`